test/circt-test/basic.mlir

```diff
@@ -31,13 +32,21 @@ verif.formal @Some.TestB {}
 // JSON-NEXT:     "wow": false
 // JSON-NEXT:   }
 // JSON-NEXT: }
-// CHECK: Attrs formal {awesome = true, engine = "bmc", offset = 42 : i64, tags = ["sby", "induction"], wow = false}
+// CHECK: Attrs formal {awesome = true, engine = "bmc", ignore = false, offset = 42 : i64, tags = ["sby", "induction"], wow = false}
 verif.formal @Attrs attributes {
     awesome = true,
     engine = "bmc",
     offset = 42 : i64,
     tags = ["sby", "induction"],
-    wow = false
+    wow = false,
+    ignore = false
+} {}
+
+// CHECK-NOT: "name": "Ignore"
+// JSON-NOT: "name": "Ignore"
+verif.formal @Ignore attributes {
+    ignore = true,
+    this = "should not appear"
```

Comment:
It might be worthwhile folding the `include-ignored.mlir` file into this one. No need to check all permutations with circt-as again. Maybe just another RUN line with `circt-test %s --include-ignored | FileCheck %s --check-prefix=CHECK-WITH-IGNORED` or similar.